roles/module_usage_tracking/README.md

```diff
@@ -0,0 +1,55 @@
+# Module Usage Tracking: list how often modules were used on an HPC cluster
+
+Lmod can track which modules are used by sending that info to the syslog each time a user executes ```module load some-module```.
+This is done with the ```load_hook``` from a custom ```SitePackage.lua```,
+which is deployed with the ```install_easybuild/``` role from the [ansible-pipelines repo](https://github.com/molgenis/ansible-pipelines).
+
+This ```module_usage_tracking``` role configures:
+
+* ```rsydlog``` on the DAIs to receive log messages tagged _ModuleUsageTracking_ and to store them in the log file.
+* ```logrotate``` on the DAIs to rotate that log file.
+
+### About the module_usage log file
+
+We do not store the module usage on each cluster node,
+but forward this info to the cluster's Deploy Admin Interface (DAI) machine,
+which writes this to the log file ```/var/log/lmod/module_usage``` in the format:
```

Comment:
```suggestion
```suggestion
but forward this info to the cluster's Deploy Admin Interface (DAI) machine.
DAI machine collects module usage log lines from all clients and together stores them inside a ```/var/log/lmod/module_usage``` log file. The format of this file is:
```
```

Reply:
Rephrased slightly differently.